.github/workflows/ci.yml

```diff
@@ -154,7 +154,8 @@ jobs:
       # actions/checkout does not let us clone into anywhere outside ${{ github.workspace }}, so we have to copy the clone...
       - name: Copy Git Repo to Dev Drive
         run: |
-          Copy-Item -Path "${{ github.workspace }}" -Destination "${{ env.UV_WORKSPACE }}" -Recurse
+          robocopy "${{ github.workspace }}" "${{ env.UV_WORKSPACE }}" /E /SL
+          if ($LASTEXITCODE -le 3) { $LASTEXITCODE = 0 }
```

Comment:
This allows copying broken symlinks.

And yes, `robocopy` uses a non-zero exit code for success.